zebrad/src/components/sync.rs

```diff
@@ -421,21 +410,21 @@ where
                 }
 
                 // Once we're below the lookahead limit, we can keep extending the tips.
-                tracing::info!(
+                debug!(
                     tips.len = self.prospective_tips.len(),
                     in_flight = self.downloads.in_flight(),
                     lookahead_limit = self.lookahead_limit,
                     "extending tips",
```

Comment:
I'd like to add a state tip height to this log so we can debug sync issues. I'll open a separate PR.

Reply:
Done in https://github.com/oxarbitrage/zebra/pull/169